docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -5,26 +5,19 @@ skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
 AddressBook *-right-> "1" UniqueParticipantList
-AddressBook *-right-> "1" UniqueTagList
 AddressBook *-down-> "1" UniqueEventList
-UniqueTagList -[hidden]down- UniqueParticipantList
-UniqueTagList -[hidden]down- UniqueParticipantList
 
-UniqueTagList *-right-> "*" Tag
 UniqueParticipantList -down-> Participant
 UniqueEventList -down-> "*" Event
 
-Participant -up-> "*" Tag
-
 Participant *--> Name
 Participant *--> Phone
 Participant *--> Email
 Participant *--> Address
 Participant *--> ParticipantId
-Participant *--> BirthDate
-Participant *--> "*" Note
-Participant *--> "*" NextOfKin
 Participant *--> "*" Event
+Participant --> "*" NextOfKin
+Participant --> BirthDate
```

Comment:
We should include 0..1 for BirthDate since birthdate is an optional field for participant